.eslintrc.js

```diff
@@ -387,6 +387,14 @@ module.exports = {
 				'jsdoc/require-param': 'off',
 			},
 		},
+		{
+			files: [ '**/@(storybook|stories)/*' ],
+			rules: {
+				// Suggested workaround for hooks used in CSF3 format stories.
+				// https://github.com/storybookjs/eslint-plugin-storybook/pull/149
+				'react-hooks/rules-of-hooks': 'off',
```

Comment:
You can simply pass a named function with a name that starts with uppercase and it will be recognized as a component. I do this in my own project. Could we try that?

I normally just go for `function Render`. See archetype-ui stories for reference, if you need to.

Reply:
Agreed. If we can avoid adding extra complexity to the ESLint configuration with a valid alternative syntax, let's do it.

Reply:
Sure, we can go with that! (Interestingly, most of the existing hooks were moved into decorator functions, which are anonymous and don't trigger the rule-of-hooks lint 😳)

Reverted in 24569a5

Reply:
Yep. Was going to say this. Just came here running a lint without the ESLint changes locally and was surprised that there were no errors.